tests/ddtrace_run_test.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+from ddtrace import tracer
+import os
+
+if __name__ == '__main__':
+    if 'DATADOG_SERVICE_ENV' in os.environ:
```

Comment:
what is this testing? shouldn't we verify it correctly calls the thing, returns output as expected, etc.